api/main.py

```diff
@@ -75,7 +75,7 @@ async def post_user(
         current_user: User = Security(get_user, scopes=["admin"])):
     """Create new user"""
     try:
-        hashed_password = auth.get_password_hash(str(password.password))
+        hashed_password = get_password_hash(password)
```

Comment:
The issue is that `get_password_hash()` is an API entry point function, so I don't think it's a good idea to call it from within another entry point function.

This should work too and would stay within the current layer:
```python
hashed_password = auth.get_password_hash(password.password.get_secret_value())
```

Reply:
Done.